charts/pelorus/subcharts/exporters/templates/_imagestream.yaml

```diff
@@ -6,5 +6,5 @@ metadata:
   labels:
```

Comment:
do we need to change anything in the create operator script to address this fix?

Reply:
yes, but now we need to unblock broken marketplace